plugins/single_plugins/idle.cpp

```diff
@@ -191,7 +228,13 @@ class wayfire_idle_singleton : public wf::singleton_plugin_t<wayfire_idle>
             return false;
         }
 
-        get_instance().toggle_idle();
+        if (toggled_off ^= 1)
```

Comment:
This would make the plugin work per-output, whereas I expect idle-inhibit shortcut to work globally?